fs/fs_zenfs.cc

```diff
@@ -1081,6 +1088,9 @@ Status NewZenFS(FileSystem** fs, const std::string& bdevname) {
           zbd_status.ToString().c_str());
     return Status::IOError(zbd_status.ToString());
   }
+  // This line is not necessary if you do not plan to use Metrics,
+  // as NoZenFSMetrics will be used by default.
+  //zbd->SetMetrics(std::make_shared<NoZenFSMetrics>());
```

Comment:
So adding `zbd->SetMetrics(std::make_shared<NoZenFSMetrics>());`doesn't make any sense? I think this comment is not necessary.